### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-03-13

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#673]
## Time

**UTC Wed 13-Mar-2019 21:00 (09:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 13-Mar-2019 14:00 (02:00 PM) |
| US / Mountain | Wed 13-Mar-2019 15:00 (03:00 PM) |
| US / Central  | Wed 13-Mar-2019 16:00 (04:00 PM) |
| US / Eastern  | Wed 13-Mar-2019 17:00 (05:00 PM) |
| London        | Wed 13-Mar-2019 21:00 (09:00 PM) |
| Amsterdam     | Wed 13-Mar-2019 22:00 (10:00 PM) |
| Moscow        | Thu 14-Mar-2019 00:00 (12:00 AM) |
| Chennai       | Thu 14-Mar-2019 02:30 (02:30 AM) |
| Hangzhou      | Thu 14-Mar-2019 05:00 (05:00 AM) |
| Tokyo         | Thu 14-Mar-2019 06:00 (06:00 AM) |
| Sydney        | Thu 14-Mar-2019 08:00 (08:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2019-03-13&iso=20190313T21
* or http://www.wolframalpha.com/input/?i=09PM+UTC%2C+Mar+13%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1gBdQeq31AGdqAPEr40HSctu2ta8ttpHMyuE3Ho4VyfQ/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* Nominating Gireesh Punathil (@gireeshpunathil) to the TSC [#26186](https://github.com/nodejs/node/issues/26186)

* Distrust Symantec root certs [14537](https://github.com/nodejs/node/issues/14537)

### nodejs/TSC

* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)

### nodejs/admin
* Checklist to address changes due to move to OpenJS Foundation [311](https://github.com/nodejs/admin/issues/311)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Rich Trott @Trott (TSC)

### Observers/Guests
* Gireesh Punathil @gireeshpunathil (observer)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by Trott** [comment]
This is the week to vote on Gireesh's nomination unless we want to delay one more week just so that he can have attended more than one meeting.

I'll also point out that it is all but certain that https://github.com/nodejs/node/pull/21351 will not land on master in time for Node.js 12 *unless* there's a TSC vote in favor of it. I personally don't think a vote in favor is likely to happen, but if someone wants to find out or just force the issue because it's the right thing to do as far as they're concerned, then putting that on the agenda is the thing for that someone to do.

**Comment by Trott** [comment]
> This is the week to vote on Gireesh's nomination unless we want to delay one more week just so that he can have attended more than one meeting.

I don't think we should delay personally. TSC meetings are getting canceled a lot lately and I think that's a great thing. (It hopefully means most things are getting taken care of in the issue tracker and we don't need to talk about them at meetings.)

I'd also be OK with canceling this meeting and breaking from tradition and having the voting-on-the-nomination happen outside of the meeting and in the issue tracker instead. We could still have a big "Hooray! Welcome!" moment about it during the Announcements section of the next meeting.

**Comment by gireeshpunathil** [comment]
Just wondering whether I am expected to join today or not (given I will be subject of a discussion)

**Comment by mhdawson** [comment]
I think it would be good to have the meeting this week.  We have a few next steps to talk about given that the Node.js and JS Foundation merger has been announced.

**Comment by mhdawson** [comment]
>This is the week to vote on Gireesh's nomination unless we want to delay one more week just so that he can have attended more than one meeting.

My opinion is that we should not delay either.

**Comment by ChALkeR** [comment]
Can we discuss https://github.com/nodejs/node/issues/14537? I added a label.

**Comment by mhdawson** [comment]
@ChALkeR added to the agenda

**Comment by Trott** [comment]
> Just wondering whether I am expected to join today or not (given I will be subject of a discussion)

@gireeshpunathil Yes, you should join!

**Comment by Trott** [comment]
> Can we discuss nodejs/node#14537? I added the label there.

@ChALkeR Are you hoping for a decision of some kind? If so, is it a decision about whether or not we should distrust those certs? Whether we should generally follow Google/Mozilla on this stuff? How we should publicize this decision? Something else?

**Comment by ChALkeR** [comment]
@Trott At this point, I am hoping for some input on the issue =). Perhaps, from the people who interacted with that thread? 

I do not hope for a final decision tomorrow, though I _suppose_ it could possibly happen with a resolution "not in 12.0, postpone" or something like that, if it would seem that that would be the best course of action, all things considered.

**Comment by mhdawson** [comment]
@MylesBorins it would be good to get an update on the Modules status. It sounded like there was consensus in the team on a proposal so it would be good to give a short overview to TSC along with next steps.  I think that can fit under the strategic initiative review but wanted to make sure you have it on your radar.

**Comment by rvagg** [comment]
@mhdawson or I should probably present briefly on the discussion we had today in Build about supported platform recommendations for Node 12. We don't quite have all the pieces in place quite yet for a PR against BUILDING.md but I think it's important enough to at least get the TSC's heads in this space and to share in our thinking.

**Comment by mcollina** [comment]
I think it's better to have the meeting, I have some  bits of private business to discuss.

**Comment by Trott** [comment]
Since I'm unsure if I'll be able to attend, but certainly we'll want a decisive number of participants in the vote on it: I'm +1 on Gireesh's nomination/membership!

**Comment by apapirovski** [comment]
I'm also unsure if I'll be able to join, or at the very least I'll be 15min late, so: +1 on Gireesh's nomination.

**Comment by targos** [comment]
I'm quite tired after following a long Modules team meeting, so:

- +1 on Gireesh's nomination!
- V8 currency: almost ready to land V8 7.3. Next step is to prepare for 7.4 and work with the V8 team so it's forward-compatible with 7.5.

**Comment by thefourtheye** [comment]
Moderation team updates:

1. A rude/offensive comment was removed (https://github.com/nodejs/moderation/issues/269)

---

cc @nodejs/tsc @nodejs/community-committee @nodejs/moderation

**Comment by Trott** [comment]
Minutes PR is at https://github.com/nodejs/TSC/pull/674 so closing this.
